rules-tests/CodeQuality/Rector/ClassMethod/ReturnTypeFromStrictScalarReturnExprRector/Fixture/skip_if_child_class_has_differing_type.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\ClassMethod\ReturnTypeFromStrictScalarReturnExprRector\Fixture;
+
+class A
+{
+    public function test()
+    {
+        return 0;
+    }
+}
+
+namespace Rector\Tests\CodeQuality\Rector\ClassMethod\ReturnTypeFromStrictScalarReturnExprRector\Fixture;
+
+class B extends A
+{
+    public function test(): string
+    {
+        $a = '1';
+        
+        return $a;
+    }
+    
+}
+
+?>
```

Comment:
This fixture must be valid PHP code. There are 2 namespaces now :)

You can extact the `A` class to `/Source` directory and only re-use it here. That way fixture has only the code it processes (changes/skips)

Last point, class name and file name should be same, to make it easier to read the fixture code. "A" will not tell me much, but "SkipForDifferentParentType" will explain a lot

Reply:
@TomasVotruba no, the part that changed is parent, not child, so it can't live in the `Source` directory, only reproduced with single file.

Reply:
Got it :wink: